magento/pwa-studio
# Release notes 5.0

## Description

This PR updates the changelog with 5.0 release content.

## Related Issue

Closes PWA-252

## Acceptance 

@jimbo
@awilcoxa

### Verification Stakeholders

@jimbo to verify items and descriptions
@awilcoxa to verify items and descriptions

### Specification



### Verification Steps

1. Open `CHANGELOG.md`
2. Verify items and descriptions are accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
